### PR TITLE
added Employee Module

### PR DESCRIPTION
**Pull request by manvithagali52**

### CODE REVIEW

**Review by sgreddy123 — APPROVED** [review]

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 1]
```diff
@@ -4,19 +4,24 @@
...
 import com.khadri.jdbc.prepare.data.driver.types.DriverTypes;
+import com.khadri.jdbc.prepare.data.employee.dao.EmployeeInsertDao;
+import com.khadri.jdbc.prepare.data.employee.processor.EmployeeDataProcessor;
 import com.khadri.jdbc.prepare.data.student.dao.StudentInsertDao;
 import com.khadri.jdbc.prepare.data.student.processor.StudentDataProcessor;
 
 public class JdbcDataDriver {
 
 	private Scanner scanner;
 	private StudentInsertDao insertDao;
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: define like stdDao

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 2]
```diff
@@ -4,19 +4,24 @@
...
+	private EmployeeInsertDao insertDao1;
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: define like empDao

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 3]
```diff
@@ -28,7 +33,7 @@ public static void main(String[] args) {
 		csvDataDriver.process();
 	}
 
-	private void process() {
+	private void process() throws Exception{
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: which exception is forcing you to throw here?

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 4]
```diff
@@ -44,10 +49,20 @@ private void process() {
 				stdProcessor.process(dataProcessId);
 			}
 
+			break;
+		case 2:
+			System.out.println("How many records you want to insert ? : ");
+			int recordCount1 = scanner.nextInt();
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: why it is recordCount1 instread recordCount ?

File: src/main/java/com/khadri/jdbc/prepare/data/employee/dao/model/Employee.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
+package com.khadri.jdbc.prepare.data.employee.dao.model;
+
+public class Employee {
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/employee/dao/model/Employee.java:3]: All the variables are not following camel case standards.

File: src/main/java/com/khadri/jdbc/prepare/data/employee/processor/EmployeeDataProcessor.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+package com.khadri.jdbc.prepare.data.employee.processor;
+
+import java.util.Scanner;
+
+import com.khadri.jdbc.prepare.data.employee.dao.EmployeeInsertDao;
+import com.khadri.jdbc.prepare.data.employee.dao.model.Employee;
+
+public class EmployeeDataProcessor {
+	    private Scanner scanner;
+		private EmployeeInsertDao insertDao1;
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/employee/processor/EmployeeDataProcessor.java]: why it is insertDao1; and why not empDao ?

File: src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeInsertDao.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+import com.khadri.jdbc.prepare.data.employee.dao.model.Employee;
+
+public class EmployeeInsertDao {
+	public static void insertData(Employee emp) throws Exception {
+		Scanner sc = new Scanner(System.in);
+		Connection con = null;
+		PreparedStatement pstmt = null;
+
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeInsertDao.java]: write separate class for getting the connection ??.
call the method and use that connection object here.

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 8]
```diff
@@ -4,31 +4,35 @@
...
-			System.out.println(each.getId() + " : " + each.getName());
+		System.out.println(each.getId() + " : " + each.getName());
 		});
 
 		JdbcDataDriver csvDataDriver = new JdbcDataDriver();
 		csvDataDriver.process();
 	}
 
-	private void process() {
+	public void process() {
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: revert back to private.

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java [hunk 9]
```diff
@@ -44,10 +48,26 @@ private void process() {
 				stdProcessor.process(dataProcessId);
 			}
 
+			break;
+		case 2:
+			System.out.println("How many records you want to insert ? : ");
```
Comment (khadri2021) [inline src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: add do you want ?

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java (hunk 8 above)
Comment (manvithagali52) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: Yes sir i revert back

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java (hunk 9 above)
Comment (manvithagali52) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: Yes sir i added

File: src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java (hunk 4 above)
Comment (manvithagali52) [reply src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java]: Yes sir i changed

**Review by khadri2021 — APPROVED** [review]

**Pull request merged**